src/messages/javascript.js

```diff
@@ -42,3 +42,10 @@ export function _methodPassedRemoteUri(method) {
 
 export const OPENDIALOG_REMOTE_URI = _methodPassedRemoteUri('openDialog');
 export const OPENDIALOG_NONLIT_URI = _nonLiteralUri('openDialog');
+
+export const EVAL = {
+  code: 'EVAL',
```

Comment:
I wonder if this code should be narrowed down to something relating to strings as args as opposed to just `EVAL` e.g. `EVAL_STRING_ARG` or something.